packetbeat/docs/packetbeat-options.asciidoc

```diff
@@ -1090,6 +1092,48 @@ at the end of the document.
 Note that limiting documents in this way means that they are no longer correctly
 formatted JSON objects.
 
+[[configuration-tls]]
+=== Capture TLS traffic
+
+++++
+<titleabbrev>TLS</titleabbrev>
+++++
+
```

Comment:
I think we need an overview of the feature that explains what it does and how it does. Some points to possibly cover:

- It captures TLS handshake data.
- Make it clear that this does not decrypt any traffic.
- It provides information about the TLS versions, ciphers, and certificates being used. It measures the time it takes for the handshake to complete.
- It computes a JA3 fingerprint of the client's TLS parameters. The fingerprint can be used to identify an application associated with the traffic. The fingerprint is independent of network addresses and certificates.

Reply:
I will leave the JA3 and handshake time for another patch so this one can be backported to 6.1.